lumigator/python/mzai/backend/backend/api/routes/jobs.py

```diff
@@ -33,12 +34,34 @@ def create_inference_job(
     return job_response
 
 
-@router.post("/evaluate/", status_code=status.HTTP_201_CREATED)
-def create_evaluation_job(
+@router.post("/annotate/", status_code=status.HTTP_201_CREATED)
+def create_annotation_job(
     service: JobServiceDep,
-    job_create_request: JobEvalCreate,
+    job_create_request: JobAnnotateCreate,
     request: Request,
-    response: Response
+    response: Response,
+) -> JobResponse:
+    # Lumigator's opinion on the best summarization model
```

Comment:
Let's use docstrings here for multiline comments https://peps.python.org/pep-0257/